games-list/src/components/AllGames.tsx

```diff
@@ -17,23 +18,37 @@ export default function AllGames({ gamesList, setList }: AllGamesProps) {
   const sortedList = gamesList
     .slice()
     .sort((a, b) => Number(a.complete) - Number(b.complete));
+  const [modal, setModal] = useState({
+    gameName: { game: "", complete: false, details: false },
+    open: false,
+  }); //Determines if modal prompt appears
```

Comment:
As a general rule, it is better to have an empty space between declarations/assignments that happen on multiple lines to improve readability.

Also, it is better to have to descriptive names for variables rather than adding comments to explain what they do. 
```suggestion
  const sortedList = gamesList
    .slice()
    .sort((a, b) => Number(a.complete) - Number(b.complete));
 
  const [openModal, setOpenModal] = useState(false)
  const [modalDetails, setModalDetails] = useState({ game: "", complete: false, details: false })
```